airbyte-integrations/connectors/source-gainsight-px/source_gainsight_px/schemas/users.json

```diff
@@ -0,0 +1,106 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Users Schema",
+  "additionalProperties": true,
+  "type": "object",
+  "properties": {
+    "aptrinsicId": {
+      "type": ["null", "string"]
+    },
+    "identifyId": {
+      "type": ["null", "string"]
+    },
+    "type": {
+      "type": ["null", "string"],
+      "enum": ["LEAD", "USER", "VISITOR", "EMPTY_USER_TYPE"]
+    },
+    "gender": {
+      "type": ["null", "string"],
+      "enum": ["MALE", "FEMALE", "OTHER", "EMPTY_GENDER"]
+    },
+    "email": {
+      "type": ["null", "string"]
+    },
+    "firstName": {
+      "type": ["null", "string"]
+    },
+    "lastName": {
+      "type": ["null", "string"]
+    },
+    "lastSeenDate": {
+      "type": ["null", "integer"],
+      "format": "int64"
+    },
+    "signUpDate": {
+      "type": ["null", "integer"],
+      "format": "int64"
+    },
+    "firstVisitDate": {
+      "type": ["null", "integer"],
+      "format": "int64"
+    },
+    "title": {
+      "type": ["null", "string"]
+    },
+    "phone": {
+      "type": ["null", "string"]
+    },
+    "score": {
+      "type": ["null", "integer"],
+      "format": "int64"
+    },
+    "role": {
+      "type": ["null", "string"]
+    },
+    "subscriptionId": {
+      "type": ["null", "string"]
+    },
+    "accountId": {
+      "type": ["null", "string"]
+    },
+    "numberOfVisits": {
+      "type": ["null", "integer"],
+      "format": "int32"
+    },
+    "location": {
+      "type": ["null", "string"]
+    },
+    "propertyKeys": {
+      "type": ["null", "array"],
+      "description": "Aptrinsic Tag Key, at least one is required",
+      "items": {
+        "type": ["null", "string"]
+      },
+      "examples": [["AP-XXXXXXXXXX-2"]]
+    },
+    "createDate": {
+      "type": ["null", "integer"],
+      "format": "int64"
+    },
+    "lastModifiedDate": {
+      "type": ["null", "integer"],
+      "format": "int64"
+    },
+    "globalUnsubscribe": {
+      "type": ["null", "boolean"]
+    },
+    "sfdcContactId": {
+      "type": ["null", "string"]
+    },
+    "lastVisitedUserAgentData": {
+      "type": ["null", "array"],
+      "items": {
+        "globalUnsubscribe": {
+          "type": ["null", "boolean"]
+        }
+      }
```

Comment:
The item is an object please declare the `type` with the `globalUnsubscribe` fields